latte/dataset/utils.py

```diff
@@ -84,3 +86,32 @@ def download(target: Union[str, pathlib.Path], _source: str, _checksum: str) ->
             "If it doesn't help, check whether the MD5 checksum are right. "
             "Note that opening a corrupted file may be a threat to your computer, due to pickle."
         )
+
+
+def split(
```

Comment:
Yes, this works :slightly_smiling_face: 

In the future, perhaps [this function](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) will be useful.